ecs_deploy/cli.py

```diff
@@ -116,7 +117,8 @@ def deploy(cluster, service, tag, image, command, env, secret, role, execution_r
             else:
                 raise
 
-        record_deployment(tag, newrelic_apikey, newrelic_appid, newrelic_region, comment, user)
+        deployment_revision = newrelic_revision if newrelic_revision else tag
```

Comment:
What do you think about moving this decision into `record_deployment()` and providing `tag` as well as `newrelic_revision` to the function? This would avoid duplication and there is still the possibility to overwrite the revision with the env var, which happens inside the function, too

Reply:
Yeah I was on the fence on this one. I'm good with that, and I'll make the change!